setup.py

```diff
@@ -350,9 +350,9 @@ def build(self):
         # Fetch all c and cpp files from the subdirs to compile.
         for library_name in self._sub_library_names:
             for extension in ["*.c", "*.cpp"]:
-                self._source_files.extend(glob.glob(
+                self._source_files.extend(sorted(glob.glob(
```

Comment:
ah, it seems so. Because my tests show that the change made the build fully reproducible (giving identical binaries in multiple independent builds in throw-away VMs)